pkg/apis/sources/v1alpha2/apiserver_types.go

```diff
@@ -76,12 +87,6 @@ type ApiServerSourceSpec struct {
 	// +required
 	Resources []APIVersionKind `json:"resources,omitempty"`
```

Comment:
Update the comment (Resources are the resources ...)

Reply:
thanks.